sdk/monitor/azquery/custom_client.go

```diff
@@ -109,47 +111,34 @@ func (e *ErrorInfo) Error() string {
 // Row of data in a table, types of data used by service specified in LogsColumnType
 type Row []any
 
-// Table - Contains the columns and rows for one table in a query response.
-type Table struct {
-	// REQUIRED; The list of columns in this table.
-	Columns []*Column `json:"columns,omitempty"`
-
-	// REQUIRED; The name of the table.
-	Name *string `json:"name,omitempty"`
-
-	// REQUIRED; The resulting rows from this query.
-	Rows []Row `json:"rows,omitempty"`
-
-	// maps column name to index for easy lookup, helper for accessing Row data
-	ColumnIndexLookup map[string]int `json:"-"`
+// ISO8601TimeInterval specifies the time range over which to query.
+// Use NewISO8601TimeInterval() for help formatting.
+// Follows the ISO8601 time interval standard with most common format being startISOTime/endISOTime.
+// Use UTC for all times.
+type ISO8601TimeInterval string
```

Comment:
Can we shorten this name?  How much value do we think there is in having the term "ISO8601" in the name?

Reply:
changed to TimeInterval in commit 
2aa0c1f